app/javascript/pages/HomePage.tsx

```diff
@@ -41,12 +41,16 @@ const HomePage = (_props: HomePageProps) => {
         secondaryButtonTitle={t("welcome.seeSaleListings")}
       />
       <div className="homepage-extra">
-        <MarkdownSection fullwidth>
-          <p>{t("welcome.newListingEmailAlert")}</p>
-          <Link className="button" href="https://confirmsubscription.com/h/y/C3BAFCD742D47910">
-            {t("welcome.signUpToday")}
-          </Link>
-        </MarkdownSection>
+        <ActionBlock
+          header={t("welcome.newListingEmailAlert")}
+          background="primary-lighter"
+          icon={<Icon size="3xl" symbol="mail" />}
```

Comment:
@software-project one other comment is that we don't want the logo on the homepage version if it's possible to not include it